assets/js/googlesitekit/modules/create-module-store.test.js

```diff
@@ -48,7 +49,7 @@ describe( 'createModuleStore store', () => {
 	beforeEach( () => {
 		registry = createRegistry();
 
-		storeDefinition = createModuleStore( MODULE_SLUG, { settingSlugs, registry } );
+		storeDefinition = createModuleStore( MODULE_SLUG, { storeName: TEST_STORE_NAME, settingSlugs, registry } );
```

Comment:
There is no `registry` argument to `createModuleStore` anymore so this can be removed 👍

Reply:
This comment is yet to be addressed.